src/core/components/questionnaires/cards/QuestionAnswer.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="card elm-width">
+    <p :class="['title', { required: isRequired }]">
+      {{ card.question }}
+      <span v-if="card.isQuestionAnswerMultipleChoiced"> ( plusieurs réponses sont possibles ) </span>
+    </p>
+    <div v-for="(qcAnswer, index) of card.qcAnswers" :key="qcAnswer._id" class="answers">
```

Comment:
en faisant ca tu ajoutes une div autour de chaque checkbok car le `v-for` repete la balise qui le porte : ici le `<div v-for="(qcAnswer, index) of card.qcAnswers"...` crée une nouvelle div par reponse possible
```
<div><q-checkbox .../></div>
<div><q-checkbox .../></div>
<div><q-checkbox .../></div>
```

ce qui est un peu etrange car on n'en pas pas besoin


je pense que tu peux simplifier comme suit : 
- une div qui englobe toutes tes reponses
- une balise template qui porte le `v-for`
```
<div class="answers">
  <template v-for="(qcAnswer, index) of card.qcAnswers" :key="qcAnswer._id">
    <q-checkbox v-if="card.isQuestionAnswerMultipleChoiced" :model-value="multipleSelectionList[index]"
      @update:model-value="updateSelectedAnswer(index)" :label="qcAnswer.text" />
    <q-radio v-else v-model="singleSelection" :val="qcAnswer._id" :label="qcAnswer.text" />
  </template>
</div>
```

Reply:
Ah oui exact ! J'avais pas en tete que ca creeait une div autour de chaque q-checkbox.